sympy/concrete/summations.py

```diff
@@ -450,8 +450,18 @@ def is_convergent(self):
             if lim_evaluated > 1:
                 return S.false
 
-        ### ------------- alternating series test ----------- ###
+        ### ----------- Raabe's test ---------------- ###
+        lim_val = Limit(sym * (sequence_term/sequence_term.subs(sym, sym + 1)
+                            - S(1)), sym, S.Infinity)
+        val = lim_val.doit()
         dict_val = sequence_term.match((-1)**(sym + p)*q)
```

Comment:
Should this line come later (in the alternating series test)?

Reply:
Tests seem to fail without this change, I am seeing what needs to be done for this.

Reply:
@jksuom @gxyd, I am trying to check whether the given series is an alternating series, if it is then Raabe's test should not be applied as it is only applicable for positive term series. Thus, even by using `simplify` the test will fail. .

Reply:
In that case why not apply the "alternating series" test earlier than Raabe's test?

Reply:
There can be a possibility that the `is_decreasing` condition is not satisfied in `alternating series` test, thus, the sequence term would be tested under `Raabe's test` and which will give an incorrect answer.

Reply:
@gxyd, one possible solution is if we place the `alternating test` before `Raabe's test` and then check if `dict_val` is `None`.

Reply:
Will it do?